primitives/runtime-interface/test-wasm/src/lib.rs

```diff
@@ -39,6 +39,11 @@ pub trait TestApi {
 		data
 	}
 
+	/// Returns 16kb data.
+	fn return_16kb() -> Vec<u32> {
+		vec![0; 4 * 1024]
```

Comment:
Can we actually use `u8` here with length `16 * 1024` ? : )

Reply:
`Vec<u8>` uses a different code path internally that doesn't leak memory before. This code path uses `SCALE` to pass the data.

Reply:
Ah good point! Worth adding a comment?

Reply:
Done